services/workflow_orchestrator.py

```
@@ -211,8 +211,26 @@ async def resolve_provider_for(
+    # ── Brain-policy first: free NVIDIA NIM always wins when configured ──
+    try:
+        import brain_policy
+        nvidia = brain_policy.resolve_free_nvidia_brain()
+        if nvidia is not None:
+            base, headers, model = nvidia
+            log.debug(
+                "resolve_provider_for(%s): using free NVIDIA brain (model=%s)", surface, model,
+            )
+            return base, headers, model
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)